src/main/java/utilities/OS.java

```diff
@@ -1,21 +1,38 @@
 package utilities;
 
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
 public enum OS {
     WINDOWS,
     LINUX,
+    MAC_OS_X,
     UNKNOWN;
 
+    private static Logger logger = LogManager.getLogger(OS.class);
+
+    /**
+     * Detects the OS of system in which this method is called.
+     * @return The detected OS. If it's unable to detect, it will return UNKNOWN.
+     */
     public static OS getOs() {
-        String os = System.getProperty("os.name").toUpperCase();
-        System.out.println(os.contains("LINUX"));
-        if(os.contains("LINUX")) {
-            return LINUX;
+        String osString = System.getProperty("os.name")
+                .toUpperCase();
+        logger.debug("OS string: " + osString);
+        OS detectedOS;
+        if(osString.contains("LINUX")) {
+            detectedOS = LINUX;
+        }
+        else if(osString.contains("WINDOWS")) {
```

Comment:
let's have a linter to take care of these formatting later.

Reply:
Agreed.